src/auth/index.ts

```diff
@@ -5,8 +5,6 @@ import { ChromeUser } from '@redhat-cloud-services/types';
 import { Store } from 'redux';
 
 import * as jwt from '../jwt/jwt';
-import { createUser, getTokenWithAuthorizationCode } from '../cognito/auth';
```

Comment:
FWIW these utils will be refactored in Martin's PR here: https://github.com/RedHatInsights/insights-chrome/pull/2702